apps/pdc-frontend/src/app/[locale]/(rootLayout)/page.tsx

```diff
@@ -79,7 +85,7 @@ const Home = async ({ params: { locale } }: { params: any }) => {
     return {
       char: letter,
       disabled: !isAvailable,
-      href: !isAvailable ? undefined : `products/alphabet/${letter.toLocaleLowerCase()}`,
+      href: !isAvailable ? undefined : `${productsSegment}/${alphabetSegment}/${letter.toLocaleLowerCase()}`,
```

Comment:
Wat zou je er van denken om een file te hebben met utility functions om URLs te genereren? Uiteindelijk zouden we die kunnen delen tussen pdc-dashboard en pdc-frontend.

Bijvoorbeeld: `createProductUrl({ locale: 'en', product: 'klacht' })`.

Eventueel met base URL argument zoals `new URL(url, base)`:

`createProductUrl({ locale: 'en', product: 'klacht' }, process.env.PDC_FRONTEND_URL)`